dds/DCPS/BuiltInTopicUtils.h

```diff
@@ -52,7 +55,10 @@ topicIsBIT(const char* name, const char* type)
   ) || (
     !ACE_OS::strcmp(name, BUILT_IN_PUBLICATION_TOPIC) &&
     !ACE_OS::strcmp(type, BUILT_IN_PUBLICATION_TOPIC_TYPE)
-  );
+  ) || (
+	!ACE_OS::strcmp(name, BUILT_IN_PARTICIPANT_LOCATION_TOPIC) &&
```

Comment:
The fuzz script is going to catch this, but there are tabs here.

Reply:
Thanks. I changed tabs to spaces.